cdc/sinkv2/tablesink/event_table_sink.go

```diff
@@ -103,6 +104,10 @@ func (e *eventTableSink[E]) GetCheckpointTs() model.ResolvedTs {
 
 func (e *eventTableSink[E]) Close() {
 	e.state.Store(pipeline.TableStateStopped)
+	for e.progressTracker.trackingCount() != 0 {
```

Comment:
Is it necessary? I think just ensure no more events will be flushed into downstreams after `Close` is called is enough.